dlp/snippets/inspect/inspect_test.go

```diff
@@ -681,3 +684,207 @@ func TestInspectTableWithCustomHotword(t *testing.T) {
 		t.Errorf("TestInspectTableWithCustomHotword got %q, want %q", got, want)
 	}
 }
+
+func TestInspectDataStoreSendToScc(t *testing.T) {
```

Comment:
Is this for a different sample?

Reply:
Yes, it's for a different sample but I think due to some unwanted commits or branch conflicts I have to add this one to this PR, Sorry for the confusion.